Source/carthage/Build.swift

```diff
@@ -25,15 +25,14 @@ public struct BuildCommand: CommandType {
 
 	/// Builds a project with the given options.
 	public func buildWithOptions(options: BuildOptions) -> ColdSignal<()> {
-		return self.openTemporaryLogFile(options)
-			.map { (stdoutHandle, temporaryURL) -> ColdSignal<()> in
+		return self.createLoggingSink(options)
+			.map { (stdoutSink, temporaryURL) -> ColdSignal<()> in
 				let directoryURL = NSURL.fileURLWithPath(options.directoryPath, isDirectory: true)!
 
 				let (stdoutSignal, schemeSignals) = self.buildProjectInDirectoryURL(directoryURL, options: options)
-				stdoutSignal.observe { data in
-					stdoutHandle.writeData(data)
-					stdoutHandle.synchronizeFile()
-				}
+				stdoutSignal
+					.map { .Next(Box($0)) }
```

Comment:
This is a temporary hack. Once we're using the latest version of RAC, everything will be `Event`s, so we can hook up the standard output signal directly to the sink.